Tests/TestHelper.php

```diff
@@ -44,9 +46,17 @@ public static function generateEncryptedPayload(RefreshTokenModel $refreshToken)
 
     public static function generateEncryptedAuthCodePayload(AuthorizationCodeModel $authCode): ?string
     {
+        $uris = $authCode->getClient()->getRedirectUris();
+        $uri = null;
+        if (\count($uris) > 0) {
+            $uri = $uris[0];
+        } else {
+            $uri = new RedirectUri('http://localhost/');
+        }
+
         $payload = json_encode([
             'client_id' => $authCode->getClient()->getIdentifier(),
-            'redirect_uri' => (string) $authCode->getClient()->getRedirectUris()[0],
+            'redirect_uri' => (string) $uri,
```

Comment:
This change seems unrelated to SF 5.

Reply:
it's a similar deal with this change as the other one. reverting these changes will cause an error for undefined offset:

1) Trikoder\Bundle\OAuth2Bundle\Tests\Integration\AuthorizationServerTest::testFailedAuthorizationWithCodeForOtherClient
Undefined offset: 0

/Users/chris/projects/project-oauth2-bundle/oauth2-bundle/Tests/TestHelper.php:51

Reply:
so i would advise keeping this change in.

Reply:
This has nothing to do with Symfony 5 and our CI is passing without this changes so I'd like to revert this for this PR. If you can submit a new PR with a test where this fails and with this fix please do so.

Reply:
ok done ....